portfolio/src/main/webapp/script.js

```diff
@@ -471,12 +474,24 @@ function centerOnMarkerAndOpenInfoWindow(placeQueryName) {
     map.setCenter(marker.position);
     const infoWindow = mapInfoWindowsDict[placeQueryName];
     infoWindow.open(map, marker);
+    closePreviouslyOpenedInfoWindow(infoWindow);
```

Comment:
I would double check whether this is trying to open the same window (i.e. clicking on the same comment twice).

Reply:
Thank you so much for the reminder! My apologies for missing this. The code now handles this situation by keeping the same window open and re-centering the map on that window, if the map has been dragged around.